server/server_test.go

```diff
@@ -50,7 +50,7 @@ func TestMain_Set_Quick(t *testing.T) {
 		defer m.Close()
 
 		// Create client.
-		client, err := pilosa.NewInternalHTTPClient(m.Server.URI.HostPort(), nil)
+		client, err := pilosa.NewInternalHTTPClient(m.Server.URI.HostPort(), pilosa.GetHTTPClient(nil))
```

Comment:
would it be better to use a default client defined at the package level here (and the two other places in this file), so that we aren't newing up lots of new clients while running tests?